action.yml

```diff
@@ -17,6 +17,10 @@ inputs:
     description: 'Display InteractiveUtils.versioninfo() after installing'
     required: false
     default: 'false'
+  project:
```

Comment:
Should this be called `julia-project` instead?

Reply:
I mirrored what other Julia GHAs did like `julia-actions/julia-runtest`: https://github.com/julia-actions/julia-runtest/blob/cee4ae9b6957e3bb443e499c39d5c29c752f80b7/action.yml#L29-L31

Reply:
I think I'll probably rename some of the internal functions to use the "Julia project" terminology though to be clearer

Reply:
I've added a comment internally